pkg/agent/endpoints/sds/handler.go

```diff
@@ -337,7 +341,15 @@ func peerWatcher(ctx context.Context) (watcher peertracker.Watcher, err error) {
 	return watcher, nil
 }
 
-func buildTLSCertificate(identity cache.Identity) (*any.Any, error) {
+func buildTLSCertificate(identity cache.Identity, defaultSVIDNameBool bool, defaultSVIDName string) (*any.Any, error) {
+	var id string
+
+	if defaultSVIDNameBool {
+		id = defaultSVIDName
+	} else {
+		id = identity.Entry.SpiffeId
+	}
```

Comment:
nit: "name" is probably a more appropriate variable name. Also, how would you feel about removing the else branch?
```suggestion
        name := identity.Entry.SpiffeId
	if defaultSVIDNameBool {
		name = defaultSVIDName
	}
```